src/com/owncloud/android/lib/resources/status/OCCapability.java

```diff
@@ -77,6 +78,7 @@ public OCCapability(){
         mVersionMicro = 0;
         mVersionString = "";
         mVersionString = "";
+        mServerName = "Nextcloud";
```

Comment:
This is the default value in the constructor. To avoid nullpointer exception we have to initialize it to something. This get's overwritten later if the field is found.